src/components/dato/Datovelger.tsx

```diff
@@ -67,16 +65,19 @@ const Datovelger: React.FC<Props> = ({
   return (
     <div className={fetSkrift ? 'datovelger-fetskrift' : 'datovelger'}>
       <FeltGruppe>
-        <Normaltekst>
-          <LocaleTekst tekst={tekstid} />
-        </Normaltekst>
+        <label htmlFor={tekstid}>
+          <Normaltekst>
+            <LocaleTekst tekst={tekstid} />
+          </Normaltekst>
+        </label>
         <div
           className={'datovelger__wrapper'}
           // onClick={(e) => e.preventDefault()}
         >
           <div className={'datepicker__container'}>
             {datobegrensning === DatoBegrensning.TidligereDatoer ? (
               <DatePicker
+                id={tekstid}
```

Comment:
Er dette trygg bruk av id? 
Hvis vi får to felter med samme tekstid på en side vil ikke react skille på de og man endrer kun innholdet på den "første". 

Eksempel; legge inn mange utdanninger/tidligere utdanninger

Reply:
Godt at du tok en titt! 

Hmm.. For å kunne knytte label-teksten med datovelgeren, så bruker man vanligvis id og da jeg testet gjennom søknadsdialogen la jeg ikke merke til noen problemer. Men problemstillingen du påpeker er jo også legit. 🤔 

@alexandraback   Har du noen innspill på hvordan man kan løse dette ellers?

Reply:
Alexandra foreslo at jeg skulle bruke en uuid! Så da løser jeg denne ved å gjøre det :)